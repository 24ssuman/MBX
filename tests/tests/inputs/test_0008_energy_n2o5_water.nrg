SYSTEM 1N2O5
MOLECULE
MONOMER n2o5
O 10.71051 8.700356 10.183187
N 11.978787 9.272378 9.334354
N 10.105936 8.159534 9.087399
O 11.654326 10.25178 8.749607
O 12.925971 8.582617 9.46727
O 9.209649 8.793659 8.651385
O 10.644033 7.179055 8.691036
ENDMON
ENDMOL
ENDSYS
SYSTEM 1N2O5 1H2O
MOLECULE
MONOMER n2o5
O 10.71051 8.700356 10.183187
N 11.978787 9.272378 9.334354
N 10.105936 8.159534 9.087399
O 11.654326 10.25178 8.749607
O 12.925971 8.582617 9.46727
O 9.209649 8.793659 8.651385
O 10.644033 7.179055 8.691036
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O 7.574532 7.734828 10.948177
H 6.874452 7.879013 10.269333
H 7.964732 8.638896 10.973631
ENDMON
ENDMOL
ENDSYS
SYSTEM 1N2O5 2H2O
MOLECULE
MONOMER n2o5
O 10.71051 8.700356 10.183187
N 11.978787 9.272378 9.334354
N 10.105936 8.159534 9.087399
O 11.654326 10.25178 8.749607
O 12.925971 8.582617 9.46727
O 9.209649 8.793659 8.651385
O 10.644033 7.179055 8.691036
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O 7.574532 7.734828 10.948177
H 6.874452 7.879013 10.269333
H 7.964732 8.638896 10.973631
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O 6.076106 10.55706 9.075092
H 6.996209 10.91874 8.919422
H 5.533026 11.352207 8.80022
ENDMON
ENDMOL
ENDSYS
SYSTEM 1N2O5 3H2O
MOLECULE
MONOMER n2o5
O 10.71051 8.700356 10.183187
N 11.978787 9.272378 9.334354
N 10.105936 8.159534 9.087399
O 11.654326 10.25178 8.749607
O 12.925971 8.582617 9.46727
O 9.209649 8.793659 8.651385
O 10.644033 7.179055 8.691036
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O 7.574532 7.734828 10.948177
H 6.874452 7.879013 10.269333
H 7.964732 8.638896 10.973631
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O 6.076106 10.55706 9.075092
H 6.996209 10.91874 8.919422
H 5.533026 11.352207 8.80022
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O 4.636807 14.033311 10.483616
H 3.750429 14.12176 10.097771
H 4.360482 13.631813 11.33143
ENDMON
ENDMOL
ENDSYS
SYSTEM 1N2O5 4H2O
MOLECULE
MONOMER n2o5
O 10.71051 8.700356 10.183187
N 11.978787 9.272378 9.334354
N 10.105936 8.159534 9.087399
O 11.654326 10.25178 8.749607
O 12.925971 8.582617 9.46727
O 9.209649 8.793659 8.651385
O 10.644033 7.179055 8.691036
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O 1.357237 15.103711 10.283038
H 1.334624 15.541963 9.409699
H 1.69214 15.821507 10.839098
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O 7.574532 7.734828 10.948177
H 6.874452 7.879013 10.269333
H 7.964732 8.638896 10.973631
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O 6.076106 10.55706 9.075092
H 6.996209 10.91874 8.919422
H 5.533026 11.352207 8.80022
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O 4.636807 14.033311 10.483616
H 3.750429 14.12176 10.097771
H 4.360482 13.631813 11.33143
ENDMON
ENDMOL
ENDSYS
