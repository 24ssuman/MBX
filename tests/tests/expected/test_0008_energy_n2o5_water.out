Energies without gradients:
system[....0]=          1.17460e+01    kcal/mol
system[....1]=          9.45722e+00    kcal/mol
system[....2]=          9.45120e+00    kcal/mol
system[....3]=          9.02363e+00    kcal/mol
system[....4]=          6.43719e+00    kcal/mol
Energies with gradients:
system[....0]=          1.17460e+01    kcal/mol

Atom             GradientX           GradientY           GradientZ
O               -3.040e+01          -2.668e+01          -2.919e+01
N                1.563e+00          -4.169e+00          -1.518e+01
N                5.396e+00           5.469e+00           4.067e+00
O               -1.217e+01           1.191e+01          -2.951e+00
O                5.169e+00          -1.223e+01           4.526e+00
O                3.770e+01          -1.338e+01           1.630e+01
O               -7.257e+00           3.908e+01           2.243e+01
system[....1]=          9.45722e+00    kcal/mol

Atom             GradientX           GradientY           GradientZ
O               -2.918e+01          -2.655e+01          -2.910e+01
N                1.915e+00          -3.417e+00          -1.596e+01
N                5.629e+00           1.113e+01           1.655e+00
O               -1.232e+01           1.047e+01          -2.197e+00
O                4.528e+00          -1.200e+01           4.446e+00
O                3.967e+01          -1.607e+01           1.682e+01
O               -6.582e+00           3.699e+01           2.219e+01
O                8.808e+00          -3.652e+01           2.361e+01
H               -1.829e+01           1.048e+01          -1.899e+01
H                5.820e+00           2.548e+01          -2.490e+00
system[....2]=          9.45120e+00    kcal/mol

Atom             GradientX           GradientY           GradientZ
O               -3.007e+01          -2.682e+01          -2.847e+01
N                2.476e+00          -2.441e+00          -1.696e+01
N                6.982e+00           1.171e+01           1.669e+00
O               -1.229e+01           9.778e+00          -1.796e+00
O                4.375e+00          -1.206e+01           4.555e+00
O                4.000e+01          -1.761e+01           1.685e+01
O               -7.222e+00           3.706e+01           2.192e+01
O                6.989e+00          -3.532e+01           2.371e+01
H               -1.716e+01           8.358e+00          -1.876e+01
H                7.441e+00           2.537e+01          -2.562e+00
O               -1.902e+01          -5.548e+01           1.985e+01
H                3.523e+01           2.084e+01          -7.044e+00
H               -1.774e+01           3.661e+01          -1.296e+01
system[....3]=          9.02363e+00    kcal/mol

Atom             GradientX           GradientY           GradientZ
O               -3.014e+01          -2.670e+01          -2.851e+01
N                2.688e+00          -2.708e+00          -1.694e+01
N                7.259e+00           1.131e+01           1.584e+00
O               -1.241e+01           9.945e+00          -1.811e+00
O                4.340e+00          -1.203e+01           4.584e+00
O                3.996e+01          -1.754e+01           1.689e+01
O               -7.459e+00           3.737e+01           2.200e+01
O                7.090e+00          -3.518e+01           2.353e+01
H               -1.725e+01           8.178e+00          -1.866e+01
H                7.465e+00           2.530e+01          -2.422e+00
O               -1.899e+01          -5.396e+01           2.046e+01
H                3.575e+01           2.028e+01          -7.348e+00
H               -1.821e+01           3.485e+01          -1.432e+01
O                3.058e+01           1.217e+01          -1.640e+01
H               -1.346e+01          -4.912e+00           6.118e+00
H               -1.723e+01          -6.344e+00           1.127e+01
system[....4]=          6.43719e+00    kcal/mol

Atom             GradientX           GradientY           GradientZ
O               -3.019e+01          -2.670e+01          -2.851e+01
N                2.763e+00          -2.695e+00          -1.696e+01
N                7.366e+00           1.129e+01           1.548e+00
O               -1.244e+01           9.933e+00          -1.801e+00
O                4.318e+00          -1.203e+01           4.588e+00
O                3.991e+01          -1.753e+01           1.691e+01
O               -7.489e+00           3.737e+01           2.201e+01
O               -8.798e+00          -2.078e+01           1.531e+01
H                2.611e+00           1.253e+01          -1.487e+01
H                3.605e+00           8.755e+00           1.555e-01
O                6.960e+00          -3.513e+01           2.356e+01
H               -1.719e+01           8.119e+00          -1.869e+01
H                7.552e+00           2.528e+01          -2.427e+00
O               -1.928e+01          -5.384e+01           2.067e+01
H                3.583e+01           2.026e+01          -7.401e+00
H               -1.796e+01           3.479e+01          -1.456e+01
O                2.836e+01           1.152e+01          -1.653e+01
H               -9.601e+00          -4.822e+00           4.714e+00
H               -1.633e+01          -6.320e+00           1.228e+01
